gh-actions/go/gotestfmt/action.yaml

```diff
@@ -0,0 +1,28 @@
+name: Install gotestfmt and our wrapper script
+description: Install gotestfmt and our wrapper script which improves the output and copies it to log files
+
+inputs:
+  version:
+    description: The version of gotestfmt to install
+    default: "latest"
+
+runs:
+  using: "composite"
+  steps:
+    - name: Install gotestfmt
+      run: go install "github.com/gotesttools/gotestfmt/v2/cmd/gotestfmt@${{ inputs.version }}"
```

Comment:
In the other actions, when we want to control the version, we are using the `tools/` subdirectory to pin the tools we are using.  I would appreciate we are using a similar mechanism than in the other actions with a fallback to latest if it’s not there.